packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/vault/ITVaultTest.java

```diff
@@ -40,17 +41,19 @@
 import javax.crypto.IllegalBlockSizeException;
 import javax.crypto.NoSuchPaddingException;
 import org.apache.commons.codec.binary.Base64;
+import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+
 public class ITVaultTest {
 
 	VaultClient client;
 	static TestEnvironment environment;
 	String time;
 	Random random;
-	final String actor = "JavaSDKTest";
+	final static public String actor = "JavaSDKTest";
```

Comment:
prettier sorts like:

```suggestion
	public static final String actor = "JavaSDKTest";
```

Reply:
@kenany yes, compiling fix it. It should be okey now